### DISCUSSION TITLE
JumpMaster에서 튜토리얼 룸으로 갈 때 월드 이동을 하는 것 같은데 해당 부분은 어떻게 하는 건가요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#607] [5. Q&A]
월드 이동에 관련한 Docs를 찾을 수가 없어서 말씀 드립니다.

월드 이동에 관련한 방법을 아시는 분 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 현재는 다른 월드 간의 이동 기능을 제공하고 있지 않습니다.
양해 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/discussions/203
